schemas/optimization.schema.tpl.json

```diff
@@ -0,0 +1,19 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/Optimization",
+    "_extends": "computation.schema.tpl.json",
+    "input": {
+        "_linkedTypes": [
+            "https://openminds.ebrains.eu/core/FileBundle",
+            "https://openminds.ebrains.eu/core/FileInstance",
+            "https://openminds.ebrains.eu/products/SoftwareVersion",
```

Comment:
same question as above. How can SoftwareVersion be an input? For ModelVersion: Is it the case that one ModelVersion is an input, is optimized by the computation (here optimization) and then another ModelVersion comes out as output?

Reply:
for SoftwareVersion, answer as above. For ModelVersion, yes.